documentation/man/features/kw-mail.rst

```diff
@@ -123,6 +125,29 @@ OPTIONS
 -l, \--list:
   Lists the settings that mail uses.
 
+\-- groups[=<group>]:
+  Initialize kw mail groups interface. 
+     
+\--create='<group>':
+  Create a new kw mail *<group>* in the database
+
+\--rename='<new_name>':
+  Rename a given group passed in the *--groups[=<group>]* the the *<new_name>*
+
+\--append='<"contact <email>">':
+  append the contacts to the given group, passed as *--groups[=<group>]*
+  or to a recent created group.
+
+\--remove='<group>':
+  Remove the given *<group>* from the database.
+  Also removes all of it's contacts associations and the contacts
+  only related to this group.
+
+\--remove-email='<email>':
+  Remove the given *<email>* from the database.
+  Also removes all of it's group associations.
+ 
```

Comment:
Don't forget to add examples of use at the end; also, create one specific section just for group management.